sklearn/linear_model/logistic.py

```diff
@@ -1422,6 +1425,9 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
         methods for logistic regression and maximum entropy models.
         Machine Learning 85(1-2):41-75.
         https://www.csie.ntu.edu.tw/~cjlin/papers/maxent_dual.pdf
+
+    Generalized Linear Model 1.1.11 Logistic Regression
```

Comment:
we don't usually add the user guide here, there's already a link to the user guide above

Reply:
Deleted the reference